pyccel/ast/variable.py

```diff
@@ -123,6 +123,7 @@ def __init__(
         precision=0,
         is_argument=False,
         is_kwonly=False,
+        is_tuple=False,
```

Comment:
Why do you need this argument? Shouldn't `is_tuple` be False unless this class is `TupleVariable`?